sdlf-utils/ingestion-examples/preprocessing/fargate-unzip/template.yaml

```diff
@@ -0,0 +1,311 @@
+# Copyright 2020 Amazon.com, Inc. and its affiliates. All Rights Reserved.
+#
+# Licensed under the Amazon Software License (the 'License').
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#   http://aws.amazon.com/asl/
+#
+# or in the 'license' file accompanying this file. This file is distributed
+# on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: "AWS::Serverless-2016-10-31"
+Description: "Data Lake ingestion artifact for pre-treating data before entering SDLF RAW Layer using Fargate tasks"
+Parameters:
+  pTaskName:
+    Type: String
+    Description: "General Task Name describing the general objective of this transformation."
+    Default: "unzipping"
+    AllowedPattern: "[a-z0-9]+?-?[a-z0-9]+?"
+  pEnv:
+    Type: String
+    Description: "Environment to deploy"
+    AllowedValues:
+      - dev
+      - test
+      - prod
+  pS3LandingBucketName:
+    Type: String
+    Description: "Landing bucket with the untreated files."
+  pS3LandingPrefix:
+    Type: String
+    Description: "Prefix of the landing bucket that will be treated"
+    Default: ""
+  pS3LandingExt:
+    Type: String
+    Description: "File extension of the files to be treated"
+    Default: ".txt.gz"
+  pS3RawBucketName:
+    Type: String
+    Description: "RAW bucket for SDLF, this bucket MUST ALREADY EXISTS"
+  pEcrUnzipDockerImage:
+    Type: String
+    Description: "ECR Image URL with the docker that will perform treating steps in the form: repository-url/image:tag"
+  pFargateTaskCPU:
+    Type: String
+    Description: "Number of cpu units used by the task"
+    AllowedValues:
+      - 256
+      - 512
+      - 1024
+      - 2048
+      - 4096
+    Default: 2048
+  pFargateTaskMemory:
+    Type: String
+    Description: "Amount of memory used by the task"
+    AllowedValues:
+      - 512
+      - 1024
+      - 2048
+      - 3072
+      - 4096
+      - 5120
+      - 6144
+      - 7168
+      - 8192
+      - 8192
+      - 30720
+    Default: 8192
+  pSubnetId:
+    Type: String
+    Description: "The name of the Subnet Id"
+  pAthenaBucketName:
+    Type: String
+    Description: "Name of the Athena Bucket"
+
+Resources:
+  ServiceLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "/aws/ecs/${AWS::StackName}-converter-task"
+    UpdateReplacePolicy: "Retain"
+    DeletionPolicy: "Retain"
+
+  #################### BUCKETS ####################
+  rS3LandingBucket:

```

Comment:
Great can we also please add a bucket policy like this one?
https://github.com/awslabs/aws-serverless-data-lake-framework/blob/b4945543f0a8f15f3516c68323822ba4fba00a3e/sdlf-foundations/nested-stacks/template-s3.yaml#L97

Reply:
Done